static/app/views/issueDetails/streamline/eventTitle.tsx

```diff
@@ -152,11 +152,6 @@ export const EventTitle = forwardRef<HTMLDivElement, EventNavigationProps>(
                   label: t('Copy Event Link'),
                   onAction: copyLink,
                 },
-                {
-                  key: 'view-json',
-                  label: t('View JSON'),
-                  onAction: downloadJson,
-                },
```

Comment:
keep?

Reply:
ill add it back for the small screens where we hide the other button